### PR TITLE
sa-3155-replace script

### PR DESCRIPTION
**Pull request by kmaranionjc**
## Issues
* [SA-3155](https://jumpcloud.atlassian.net/browse/3155) - SA-3155

## What does this solve?
Replaces the current Report systems to users script to a script that returns more information/data.
## Is there anything particularly tricky?
n/a
## How should this be tested?
1. Run the script and validate all fields

[SA-3155]: https://jumpcloud.atlassian.net/browse/SA-3155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: _posts/2023-02-14-Report-Systems-To-Users.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+A CSV report of systems and their user associations (inactive and active). This script will generate a "Report-SystemToUserAssociation.csv" file relative to the location this script is saved and ran.
+
+### Basic Usage:
+
+* Install the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Using-the-JumpCloud-PowerShell-Module)
+* Save the contents of the script example to a file on a system.
+  * EX: ~/Report-Systems_To_User_Associations.ps1
+  * Edit the the `Global Configuration - Start` section with your APIKey, report file name, and user associations
+* In a PowerShell terminal window run:
+  * `~/Report-Systems_To_User_Associations.ps1`
```
Comment (jworkmanjc) [inline _posts/2023-02-14-Report-Systems-To-Users.md:20]: I think just adding a section here to state that the report file name will be saved in the current working directory would be helpful for some folks

**Review by jworkmanjc — CHANGES_REQUESTED** [review]
Tested, works and the only thing I think we need to change is just to replace the existing file, not create a new one. I'm sure support folks have this link saved somewhere and I'd rather see us replace the existing post, and add a note that it was updated to include more functionality.

**Review by kmaranionjc — COMMENTED** [review]
Added the note to Additional Information

**Review by jworkmanjc — APPROVED** [review]
Awesome, thanks for the changes, looks good to me 👍

**Review by gweinjc — APPROVED** [review]
Tested good

**Pull request merged**